wg-meetings/2016-04-21.md

```diff
@@ -0,0 +1,79 @@
+# Node.js API WG - Minutes April 21 2016
+
+## Agenda
+
+* Stand up
+* Actions from last meeting
+* Review  discussion/plan from recent vm neutrality face to face
+* Open issues review
+
+## Present
+
+* Michael Dawson @mhdawson
+* Arunesh Chandra @aruneshc
+* Stefan Budeanu @stefanmb
+* Josh Gavant 
+* Jianchun
+* Alexis 
```

Comment:
Fill in other names?

Josh Gavant @joshgav
Jianchun Xu @jianchun
Alexis Campailla @orangemocha